graph/types/schema.graphqls

```diff
@@ -52,6 +52,7 @@ input PostContentInput {
   mentionedEntities:[String!],
   quotedPostID: ID
   media: MediaInput,
+  mediaID: ID,
```

Comment:
This is contained in the `mediaInput` object. We shouldn't have it here too, I don't think.

Reply:
this is correct. this was more of a testing situation and we don't need mediaInput anymore as we are fetching the record by ID.

I'll delete MediaInputNow